From df19648a151bf1361f5046d2e8ae4e86c651fd45 Mon Sep 17 00:00:00 2001
From: Hyun Kwon <hyun.kwon@xilinx.com>
Date: Tue, 23 Jan 2018 17:46:37 -0800
Subject: [PATCH 0247/1851] drm: xlnx: Kconfig: Select dpdma for ZynqMP DP

commit 6c8430e8ffd6068dfe468cb19774751a087c5eb3 from
https://github.com/Xilinx/linux-xlnx.git

Select the dedicated DMA engine for ZynqMP DP in Kconfig.

Signed-off-by: Hyun Kwon <hyun.kwon@xilinx.com>
Signed-off-by: Michal Simek <michal.simek@xilinx.com>
State: pending
Signed-off-by: Yaliang Wang <Yaliang.Wang@windriver.com>
---
 drivers/gpu/drm/xlnx/Kconfig | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/gpu/drm/xlnx/Kconfig b/drivers/gpu/drm/xlnx/Kconfig
index 041df1740b56..d6101a421aa7 100644
--- a/drivers/gpu/drm/xlnx/Kconfig
+++ b/drivers/gpu/drm/xlnx/Kconfig
@@ -33,7 +33,7 @@ config DRM_XLNX_BRIDGE_DEBUG_FS
 config DRM_ZYNQMP_DPSUB
 	tristate "ZynqMP DP Subsystem Driver"
 	depends on ARCH_ZYNQMP && OF && DRM_XLNX && COMMON_CLK
-	select DMA_ENGINE
+	select XILINX_DPDMA
 	select GENERIC_PHY
 	help
 	  DRM KMS driver for ZynqMP DP Subsystem controller. Choose
-- 
2.31.1

